src/commands/force/user/create.ts

```diff
@@ -181,11 +192,17 @@ export class UserCreateCommand extends SfdxCommand {
         .getConnection()
         .query(`SELECT id FROM profile WHERE name='${name}'`);
       defaultFields.profileId = response.records[0].Id;
-      delete defaultFields['profileName'];
     }
 
     // the file schema is camelCase while the cli arg is no capitialization
-    if (defaultFields['generatePassword'] || defaultFields['generatepassword']) {
+    // the flag and file can pass as a string or a proper boolean
+    // we can check only for existence because it could be set to false or 'false'
+    if (
+      defaultFields['generatePassword'] === 'true' ||
+      defaultFields['generatepassword'] === 'true' ||
+      defaultFields['generatePassword'] === true ||
+      defaultFields['generatepassword'] === true
+    ) {
       // standardize on 'generatepassword'
       defaultFields['generatepassword'] = 'true';
```

Comment:
I would love to just set this to `true` here instead of the string. It would clean up the if statement in the other method which is really the only place that users this right? The only thing is the --json output. Would this consider a breaking change... 

Also, if we are standardizing on `generatepassword` should we remove `generatePassword`? Otherwise, since we no longer delete in the other method, we could get two of these in the json output.

We could standardize on lowercase as string and camel as boolean. 

```
// standardize
defaultFields['generatepassword'] = 'true';
defaultFields['generatePassword'] = true;
```

It would allow us to use boolean in code but not break json output. Thoughts?